custom_connectors/custom_auth/trackvia_connector.rb

```diff
@@ -0,0 +1,769 @@
+      end,
+
+      output_fields: lambda { |object_definitions|
+        object_definitions['response_record']
+      },
+
+      sample_output: lambda { |_connection, input|
+        call(:get_fields_sample_output, view_id: input['view_id'])
+      }
+    },
+    # DELETE requests
+    delete_record: {
+      description: "Delete <span class='provider'> record</span> in " \
+      "<span class='provider'>TrackVia</span>.",
+      help: 'Delete a record in TrackVia',
+      config_fields: [
+        {
+          name: 'app_name',
+          label: 'App',
+          type: 'string',
+          control_type: 'select',
+          pick_list: 'apps',
+          optional: false,
+          change_on_blur: true,
+          hint: 'Select a TrackVia application from the list above'
+        },
+        {
+          name: 'view_id',
+          label: 'View',
+          type: 'integer',
+          control_type: 'select',
+          pick_list: 'views',
+          pick_list_params: { app_name: 'app_name' },
+          optional: false,
+          hint: 'Select an available view from the list above.'
+        },
+        {
+          name: 'id',
+          type: 'integer',
+          control_type: 'number',
+          optional: false
+        }
+      ],
+
+      execute: lambda do |_connection, input|
+        delete("/openapi/views/#{input['view_id']}" \
+        "/records/#{input['id']}")
+          .after_error_response(/.*/) do |_code, body, _header, message|
+          error("#{message} : #{body}")
+        end
+      end
+    },
+    delete_all_records_in_view: {
+      description: "Delete all <span class='provider'>" \
+      'records</span> from a view in ' \
+      "<span class='provider'>TrackVia</span>.",
+      help: 'Delete all records from a view in TrackVia',
+      config_fields: [
+        {
+          name: 'app_name',
+          label: 'App',
+          type: 'string',
+          control_type: 'select',
+          pick_list: 'apps',
+          optional: false,
+          change_on_blur: true,
+          hint: 'Select a TrackVia application from the list above'
+        },
+        {
+          name: 'view_id',
+          label: 'View',
+          type: 'integer',
+          control_type: 'select',
+          pick_list: 'views',
+          pick_list_params: { app_name: 'app_name' },
+          optional: false,
+          hint: 'Select an available view from the list above.'
+        }
+      ],
+
+      execute: lambda do |_connection, input|
+        delete("/openapi/views/#{input['view_id']}/records/all")
+          .after_error_response(/.*/) do |_code, body, _header, message|
+            error("#{message} : #{body}")
+          end
+      end
+    }
+  },
+  triggers: {
+    new_record: {
+      description: "New <span class='provider'>" \
+      'record</span> added to view in ' \
+      "<span class='provider'>TrackVia</span>.",
+      help: 'Triggers whenever a record is created and ' \
+      'added to a specified TrackVia view.',
```

Comment:
How about this?
```
Triggers whenever a record is created and ' \
      added to view in TrackVia.
```

Reply:
Talked with product team to fix labeling, please see changed code.